interfaces/apparmor/export_test.go

```diff
@@ -30,3 +32,21 @@ func MockProfilesPath(t *testutil.BaseTest, profiles string) {
 		profilesPath = realProfilesPath
 	})
 }
+
+// MockTemplate replaces apprmor template.
+//
+// NOTE: The real apparmor template is long. For testing it is convenient for
+// replace it with a shorter snippet.
+func MockTemplate(fakeTemplate string) (restore func()) {
+	orig := defaultTemplate
+	defaultTemplate = fakeTemplate
+	return func() { defaultTemplate = orig }
+}
+
+// MockExternalCommands mocks and returns MockCmd for each of the external
+// commands used in this package.
+func MockExternalCommands(c *check.C) map[string]*testutil.MockCmd {
```

Comment:
So that we can mock apparmor parser away in a consistent way. Upcoming udev backend also uses the same method. I can just mock it directly in tests if you prefer it that way.